modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -58,10 +58,37 @@ Card Border:: Specifies the card border thickness.
 Card Border Radius:: Specifies the card border radius. A larger radius makes the cards look more rounded.
 Card Shadow:: Specifies the shadow beneath each card.
 Message If Empty:: Shows a message if the carousel has no APIs. Default is `There are no APIs available right now.`
-Filter Condition (Advanced):: Limits the APIs shown in the carousel to those matching a standard Salesforce Object Query Language (SOQL) search. SOQL searches use package names, field names such as "API name" or "API description," and comparison of strings to the values of those fields.
+Filter Condition (Advanced):: Causes the carousel to show only APIs that match the given search.
 API Details Button Label:: Enables you to set the API Details button label. Default is *API Details*.
 Learn More Button Label:: Enables you to set the Learn More button label. Default is *Learn More*.
 
+=== Filter Conditions
+
+The API Carousel component parameter *Filter Condition (Advanced)* causes the carousel to filter out the APIs that do not match the condition and show only the APIs that match. A condition is a standard search in Salesforce Object Query Language (SOQL).
+
+You can search these fields:
+
+* `acm_pkg_dev__Icon__c`
```

Comment:
there seems to be a problem when markdown renders this text, because of the "_" - when I see it rendered in github, I read "acm_pkg_devIconc". Please double-check if/how to escape _ and __ to make sure the field names are displayed correctly.

Reply:
You're right. Sorry about that! Fixed.